pages/index.js

```diff
@@ -3,77 +3,45 @@ import Languages from "@/components/languages/Languages";
 import Themes from "@/components/themes/Themes";
 import Difficulty from "@/components/difficulty/Difficulty";
 import { ContainerMain } from "@/components/styledComponents/Container.styled";
+import { useRouter } from "next/router";
 
 export default function HomePage() {
   const [config, setConfig] = useState({
     language: "",
     theme: "",
     difficulty: "",
   });
-
-  // call from openai
-  const [answer, setAnswer] = useState(null);
-  const [loading, setLoading] = useState(false);
+  const router = useRouter();
 
   function handleNewLanguage(language) {
-    setConfig({ ...config, language });
+    setConfig((prevState) => ({ ...prevState, language }));
   }
+
   function handleNewTheme(theme) {
-    setConfig({ ...config, theme });
-  }
-  function handleNewDifficulty(difficulty) {
-    setConfig({ ...config, difficulty });
+    setConfig((prevState) => ({ ...prevState, theme }));
   }
 
-  async function fetcher(data) {
-    setLoading(true);
-
-    try {
-      const response = await fetch("/api/gpt", {
-        method: "POST",
-        headers: { "Content-Type": "application/json" },
-        body: JSON.stringify(data),
-      });
-      if (response.ok) {
-        const result = await response.json();
-        setAnswer(result);
-        console.log(result);
-      } else {
-        console.error("Bad Response");
-      }
-    } catch (error) {
-      console.error(error);
-    } finally {
-      setLoading(false);
-    }
+  function handleNewDifficulty(difficulty) {
+    setConfig((prevState) => ({ ...prevState, difficulty }));
   }
 
   function handleSubmit(event) {
     event.preventDefault();
-    fetcher(config);
+    router.push({
+      pathname: "/screening",
+      query: config,
+    });
   }
 
   return (
-    <>
-      <ContainerMain>
-        <h2> tongue twister</h2>
-        <Languages config={config.config} onLanguage={handleNewLanguage} />
-        <Themes theme={config.theme} onTheme={handleNewTheme} />
-        <Difficulty
-          difficulty={config.difficulty}
-          onDifficulty={handleNewDifficulty}
-        />
-        <form onSubmit={handleSubmit}>
-          <button type="submit" disabled={loading}>
-            {loading ? "Loading..." : "Get Answer"}
-          </button>
-        </form>
-      </ContainerMain>
-      <div>
-        {loading && <p>loading...</p>}
-        {/* answer from openai */}
-        {answer && <p>{answer.answer.content}</p>}
-      </div>
-    </>
+    <ContainerMain>
+      <h2>Virelangue</h2>
+      <Languages onLanguage={handleNewLanguage} />
+      <Themes onTheme={handleNewTheme} />
+      <Difficulty onDifficulty={handleNewDifficulty} />
+      <form onSubmit={handleSubmit}>
+        <button type="submit">Obtenir une réponse</button>
+      </form>
```

Comment:
it makes semantically more sense for the inputs to be part of the form:

```suggestion
      <form onSubmit={handleSubmit}>
           <Languages onLanguage={handleNewLanguage} />
           <Themes onTheme={handleNewTheme} />
           <Difficulty onDifficulty={handleNewDifficulty} />
           <button type="submit">Obtenir une réponse</button>
      </form>
```